package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "jam",
+  "version": "1.0.0",
+  "description": "3 tasks of code jam",
+  "main": "codejam.js",
+  "scripts": {
+    "test": "mocha test/test.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eveshad/rsschool-codejam/tree/codejam-3"
+  },
+  "author": "eveshad <ewashadevsk@gmail.com>",
+  "license": "ISC",
+  "devDependencies": {
+    "mocha": "^5.2.0"
```

Comment:
Even though the **.eslintrc** file is added to the project. The **eslint** dependency  and other related are not added to the the dependencies.
It means that if I don't have the eslint installed on my machine, the linter will NOT work for me even if I run `npm install`. Because you haven't listed it in your project's dependencies.